.changeset/moody-ducks-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
```

Comment:
Scaffolder backend is on version 1.0 so new features should be `minor` :)

Reply:
Gotcha!